src/collector.js

```diff
@@ -76,6 +77,10 @@ export default class Collector {
     return false;
   }
 
+  recordScannedFile(filename) {
+    this.scannedFiles.push(filename);
```

Comment:
Thinking aloud, maybe an object with filename keys and value of a list containing all scanners used could work and keep it reasonably compact. `Object.keys(obj)` would then get the full file list easily too.

Very crude example:

```js
{ 
  'foo/bar.js': ['js', 'filetype'],
  'bar.css': ['css', 'filetype'],
  'someRandomFile': ['filetype'],
}
```